pkg/console/subresource/consoleserver/config_builder.go

```diff
@@ -0,0 +1,143 @@
+package consoleserver
+
+import (
+	operatorv1 "github.com/openshift/api/operator/v1"
+	"github.com/openshift/console-operator/pkg/api"
+	yaml "gopkg.in/yaml.v2"
+	"k8s.io/klog"
+)
+
+const (
+	clientSecretFilePath    = "/var/oauth-config/clientSecret"
+	oauthEndpointCAFilePath = "/var/run/secrets/kubernetes.io/serviceaccount/ca.crt"
+	// serving info
+	certFilePath = "/var/serving-cert/tls.crt"
+	keyFilePath  = "/var/serving-cert/tls.key"
+)
+
+// ConsoleServerCLIConfigBuilder
+// Director will be DefaultConfigMap()
+//
+// b := ConsoleYamlConfigBuilder{}
+// return the default config value immediately:
+//   b.Config()
+// set all the values:
+//   b.SetHost(host).SetLogoutURL().setBrand().setDocURL().SetAPIServerURL().SetProductName().SetCustomLogoFile().Config()
```

Comment:
The method names in docs are not reflecting the real names.

Reply:
I'll remove or update. Was looking into idiomatic go, the word "set" is not typically used.